src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -837,6 +873,23 @@ type Coordinates {
     lon: Float
 }
 
+input SearchWindowOptions @oneOf {
+    departure: SearchWindow
+    arrival: SearchWindow
+}
+
+input SearchWindow {
+    dateTime: DateTime
+    timeZone: TimeZone
```

Comment:
We talking about this at the last meeting and came to the conclusion that we don't want the timezone separately, but rather use a full ISO8601 datetime with offset.

Reply:
I though about it, and for now I think requiring the offset is good. But, it is not so obvious as I first though. There is a long discussion here: https://github.com/opentripplanner/OpenTripPlanner/issues/2602

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/910717a739f12e99b3a445c338fd24a8e4b73637